### PR TITLE
[CWS] Run CWS functional tests on Amazon Linux 2 

### PR DESCRIPTION
**Pull request by lebauce**
### What does this PR do?

Run CWS functional tests on Amazon Linux 2

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] The appropriate `team/..` label has been applied, if known.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided if the `qa/skip-qa` label is not applied.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

**Review by paulcacheux — APPROVED** [review]

File: test/kitchen/drivers/ec2-driver.yml [hunk 1]
```diff
@@ -77,7 +77,7 @@ platforms:
     <% end %>
     image_id: <%= platform[1] %>
     block_device_mappings:
-      - device_name: /dev/sda1
+      - device_name: /dev/xvda
```
[the remark this answers is not in the document]
Comment (brycekahle) [reply test/kitchen/drivers/ec2-driver.yml:80]: I'm running our `arm64` tests, which use EC2, as a final check

**Review by brycekahle — APPROVED** [review]

File: go.mod [hunk 2]
```diff
@@ -67,7 +67,7 @@ require (
 	github.com/DataDog/ebpf-manager v0.0.0-20220223090948-29871766cdd0
 	github.com/DataDog/gohai v0.0.0-20220121142033-4006c7a9c63d
 	github.com/DataDog/gopsutil v0.0.0-20220308095538-d086941833e3
-	github.com/DataDog/nikos v1.7.4
+	github.com/DataDog/nikos v1.7.5-0.20220301203122-e057b84453ad
```
Comment (mx-psi) [inline go.mod:70]: Is this related to the functional tests? If so, should this have a changelog note?

**Review by mx-psi — APPROVED** [review]
